mubench.reviewsite/templates/review.phtml

```diff
@@ -106,34 +106,26 @@ $origin_path = $origin_path ? $origin_path : $pathFor('experiment.detector', arr
             <?php if($experiment->id !== 2) : ?>
                 <tr><td><b>Misuse:</b></td><td>misuse '<?= $markdown_parser->text($misuse->misuse_muid); ?>'</td></tr>
             <?php endif; ?>
-            <?php if($user || $misuse->misuse_tags) : ?>
+            <?php $misuse_tags = $misuse->getReview($reviewer)->tags; ?>
+            <?php if($user || $misuse_tags->count() > 0) : ?>
                 <tr>
                 <td><b>Tags:</b></td>
                 <td>
                     <div id="misuse-tags" style="margin:0;padding:0;display:inline-block">
-                    <?php if($misuse->misuse_tags): ?>
-                    <?php foreach($misuse->misuse_tags as $tag) : ?>
+                    <?php if($misuse_tags): ?>
+                        <?php foreach($misuse_tags as $tag) : ?>
                             <div class="misuse-tag" style="background-color:<?= htmlspecialchars($tag->color) ?>;color:<?= htmlspecialchars($tag->getFontColor()) ?>;">
                                 <span id="<?= htmlspecialchars($tag->name) ?>"><?= htmlspecialchars($tag->name) ?></span>
                                 <?php if($user) : ?>
-                                    <button onclick="removeTag('<?= $pathFor('tag.remove', array('experiment_id' => $experiment->id,
-                                        'detector_muid' => $detectorPathId($detector),
-                                        'project_muid' => $misuse->getProject(),
-                                        'version_muid' => $misuse->getVersion(),
-                                        'misuse_muid' => $misuse->misuse_muid,
-                                        'tag_id' => $tag->id)) ?>', '<?= $misuse->id ?>', this.parentElement)"style="border:none;outline:none;background-color:<?= htmlspecialchars($tag->color) ?>;margin:0;padding:0"><i class="fa fa-trash"></i></button>
+                                    <button onclick="removeTag(this.parentElement)"style="border:none;outline:none;background-color:<?= htmlspecialchars($tag->color) ?>;margin:0;padding:0"><i class="fa fa-trash"></i></button>
```

Comment:
This is the tag display at the top of the page, right? What happens if I click remove here? The tag vanishes, but since this is not immediately synced to the server, it gets lost with the next action in any case, doesn't it? Shouldn't this be display-only?